QGL/BasicSequences/RB.py

```diff
@@ -87,7 +89,7 @@ def create_RB_seqs(numQubits,
 
     return seqs
 
-def SingleQubitRB(qubit, seqs, purity=False, showPlot=False, add_cals=True):
+def SingleQubitRB(qubit, seqs, cliff_type='std', purity=False, showPlot=False, add_cals=True):
```

Comment:
Does this mean we can now depricate/delete the other Specialized RB functions?

Reply:
Yeah, I left them in with a warning that they are deprecated and we may delete them at some point for compatibility.

Reply:
👍🏻